### DISCUSSION TITLE
Karpenter preferred module

### DISCUSSION THREAD

**Discussion by cdenneen** [aws-ia/terraform-aws-eks-blueprints-addons#362] [General]
Question will the karpenter here be the preferred way to install and replace the nested module karpenter inside the eks module or will the eks one become the preferred one and the one here would be deprecated? Since both are maintained by AWS I would expect one to become the long term replacement. Would guess blueprints would win out and the nested ones inside eks would be migrated to use these ones but unclear as to the current roadmap.

**Comment by askulkarni2** [comment]
See https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/discussions/351.
